pygmt/src/grdfill.py

```diff
@@ -17,7 +17,9 @@
 @use_alias(
     A="mode",
     G="outgrid",
+    N="node",
```

Comment:
Is `node` a good alias name? [GMT.jl](https://www.generic-mapping-tools.org/GMT.jl/dev/#GMT.grdfill) uses `nodata`.

Reply:
See https://github.com/GenericMappingTools/gmt/issues/5246. So **-N** option is not available in GMT 6.2.0rc1.

Reply:
removed in 2b8c131